modules/helfi_base_content/helfi_base_content.install

```diff
@@ -254,3 +254,13 @@ function helfi_base_content_update_9003() : void {
     ->set('selected_langcode', 'site_default')
     ->save();
 }
+
+/**
+ * Implements hook_install().
+ */
+function helfi_base_content_update_9004() : void {
+  $config_factory = Drupal::configFactory();
+
+  // Remove the default frontpage view.
+  $config_factory->getEditable('system.menu.branding-navigation')->delete();
```

Comment:
This deletes the menu, but the block remains active which results in the "This block is broken and missing" message being displayed for logged in users.

Reply:
Good catch here - I actually added the removal of the blocks as well since they shouldn't be left behind. There shouldn't be this issue after removing them as well 🙏